angular/src/app/workflows/workflow-detail/workflow-detail.component.html

```diff
@@ -0,0 +1,7 @@
+<ddap-header [backLink]="['../../../../..']" label="Workflow Detail"></ddap-header>
```

Comment:
Please see comment for https://github.com/DNAstack/ddap-frontend/pull/222/files#diff-7a5262b16d0b8befeb2b597721d5fd53R11